lib/services/auth_service.dart

```diff
@@ -196,19 +211,23 @@ class AuthService {
       yield RemoveCurrentPage();
     } on FirebaseAuthException catch (e) {
       yield UpdateEmailAuthOptionsPage(step: AuthStep.waitingForInput);
-      yield AddProblem.from(
-        message: e.message.toString(),
-        type: ProblemType.emailSignIn,
-        info: BuiltMap({'code': e.code}),
+      yield AddProblem(
+        problem: EmailSignInProblem.by(
```

Comment:
I think this should be EmailSignUpProblem?  I missed this first time through I guess :-)

Reply:
Another good catch! Fixed 👍